src/pages/guides/changelog/index.md

```diff
@@ -20,8 +20,18 @@ contributors:
   - https://github.com/amandahuarng
 ---
 
+<InlineAlert variant="warning" slots="header, text1, text2" />
+
+IMPORTANT: Deprecation Warning
+
+This version of the docs and SDK will be sunsetted later this year. As a result, we are no longer approving new submissions that integrate with v2.
+
+Instead, we would love for you to start testing the new version of the SDK. To join the private beta and get early access to documentation, please fill out [this form](https://airtable.com/shryiOk1VwoWxUCZs?prefill_Platform=Adobe%20Express%20Embed%20SDK&hide_Platform=true).
```

Comment:
we would love for you to start - Suggested Edit - 
`We would like you to start integrating new  version of the SDK`